x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityRestIT.java

```diff
@@ -83,7 +94,12 @@ public class RemoteClusterSecurityRestIT extends AbstractRemoteClusterSecurityTe
                         {
                           "search": [
                             {
-                                "names": ["index*", "not_found_index", "shared-metrics"]
+                                "names": ["employees", "index*", "not_found_index", "shared-metrics"]
```

Comment:
Need `employees` here since the key otherwise doesn't have the right access for the test case.